tests/Behat/FeatureContext.php

```diff
@@ -187,6 +187,83 @@ protected function getLangcodeByName(string $language_name): string {
     throw new \Exception("Language name '$language_name' is not valid.");
   }
 
+  /**
+   * Visit a edit page by node title.
+   *
+   * @When I visit node :arg1 edit page
+   */
+  public function iVisitNodeEditPage($arg1) {
+    $node = $this->getNodeByTitle($arg1);
+    $this->visitPath("/node/{$node->id()}/edit");
+  }
+
+  /**
+   * Checks, that form field has specified value.
+   *
+   * @Then the :field field contains :value
+   */
+  public function theFieldContains($field, $value) {
+    $node = $this->getSession()->getPage()->findField($field);
+    $actual = $node->getValue();
+
+    if (strpos($actual, $value) === FALSE) {
+      throw new \Exception(sprintf('Field %s does not contain %s.', $field, $value));
+    }
+  }
+
+  /**
+   * Attempts to find a button in a table row containing giving text.
+   *
+   * @When I press :button in the :rowText row
+   */
+  public function iPressInTheRow($button, $rowText) {
```

Comment:
Missing type hints and code sniffing shouldn't have passed here, as camel case is not allowed for method parameters. Check if that actually runs on these classes too.